src/modules/map/components/infoButton/InfoButton.js

```diff
@@ -68,6 +57,18 @@ export class InfoButton extends BaElement {
         `;
 	} 
 
+	extractState(store) {
+		const { map: { zoom, position } } = store;
+		return { zoom, position };
+	}
+
+	onStateChanged() {
+		var popup = this.shadowRoot.getElementById('info-popup');
+		if (popup.isOpen()) {
+			popup.closePopup();
+		}
```

Comment:
We can omit the if-clause here (also near line 35)
(have also a at your code coverage report for this file: coverage/modules/map/components/infoButton/index.html)